src/libs/lv_conf.h

```diff
@@ -1,95 +1,43 @@
-/**
```

Comment:
This is a LVGL-related change, not NimBLE-related change.

Reply:
There is no actual difference 😂 I was just removing comments for fun. Will be defaulting it